### PR TITLE
Test: APPLICS-358-RTHYS-change-answers

### PR DESCRIPTION
**Pull request by willadams332**
## Describe your changes

- Changes to the RTHYS flow
- Changes to how we store case data
- Adding custom commands to navigation
- Removed unused methods in test
- Fixed failing tests

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [X] Other (please explain in the description section above)

## Checklist before requesting a review

- [X] I have performed a self-review of my own code
- [X] I have double checked this work does not include any hardcoded secrets or passwords
- [X] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [X] I have referenced the ticket number above
- [X] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/e2e_tests/cypress/e2e/register-as-an-agent.cy.js [hunk 1]
```diff
@@ -1,110 +1,192 @@
...
+			cy.fixture('caseData').then((data) => {
+				caseData = data;
+				caseData.useOrgName = data.useOrgName;
+			});
+
+			cy.navigateAndSearch('Front Office Auto Test');
+		});
+
+		it('Should click on register to have your say link and assert page content', () => {
+			basePage.clickProjectInformationMenuLink('have-your-say');
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/e2e/register-as-an-agent.cy.js:32]: should this be register-have-your-say?

File: packages/e2e_tests/cypress/e2e/examinaton-timetable-organisation-submission.cy.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 const basePage = new BasePage();
 
-before(() => {
-	cy.clearCookies();
-});
-
 describe('User registers as an organisation to have their say against the examination timetable', () => {
+	before(() => {
+		cy.clearCookies();
+		cy.navigateAndSearch('Front Office Auto Test');
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/e2e/examinaton-timetable-organisation-submission.cy.js:10]: Is this going to work?  I checked on test and that search on project page didn't seem to do it.  May be wrong though.

Comment (willadams332) [reply packages/e2e_tests/cypress/e2e/examinaton-timetable-organisation-submission.cy.js:10]: I added this method to all tests as we were navigating directly to the project page, which would not replicate a end to end journey. all tests have passed with this new method or are you specifically calling out this test?

File: packages/e2e_tests/cypress/e2e/register-as-an-agent.cy.js (hunk 1 above)
Comment (willadams332) [reply packages/e2e_tests/cypress/e2e/register-as-an-agent.cy.js:32]: with this method, I created a switch case for the side menu links, I called the case 'have-your-say' but its going to 
the register-have-your-say journey. 

`case 'have-your-say':
					cy.get('a[href*="/register-have-your-say"]').click();
					break;`

I can change this in my next PR.

**Review by Philip-Philippou1 — DISMISSED** [review]

**Pull request merged**